test/minikube/minikube_base_collector_test.go

```diff
@@ -0,0 +1,200 @@
+// +build long
+
+package minikube
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/nuodb/nuodb-helm-charts/test/testlib"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	v12 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const FILE_PLUGIN_CONFIGMAP = `
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nuocollector-insights-file
+  labels:
+    nuodb.com/nuocollector-plugin: insights
+data:
+  file.conf: |-
+    [[outputs.file]]
+    files = ["stdout"]
+    data_format = "influx"
+`
+
+func createOutputFilePlugin(t *testing.T, namespaceName string) {
+	kubectlOptions := k8s.NewKubectlOptions("", "", namespaceName)
+	k8s.KubectlApplyFromString(t, kubectlOptions, FILE_PLUGIN_CONFIGMAP)
+	testlib.AddTeardown(testlib.TEARDOWN_COLLECTOR, func() { k8s.KubectlDeleteFromStringE(t, kubectlOptions, FILE_PLUGIN_CONFIGMAP) })
+	// Wait a bit so that the nuocollector-config sidecar refresh the Telegraf configuration
+	time.Sleep(3 * time.Second)
+}
+
+func checkMetricsLine(t *testing.T, namespaceName string, podName string,
+	expectedLine string, minOccurances int) bool {
+	count := testlib.GetRegexOccurrenceInLog(t, namespaceName, podName, expectedLine, &v12.PodLogOptions{Container: "nuocollector"})
+	if count >= minOccurances {
+		t.Logf("Found %d occurances of '%s' in pod %s log", count, expectedLine, podName)
+		return true
+	}
+	return false
+}
+
+func verifyCollectionForAdmin(t *testing.T, namespaceName string, app string) {
+	options := k8s.NewKubectlOptions("", "", namespaceName)
+	pods := k8s.ListPods(t, options, metav1.ListOptions{
+		LabelSelector: fmt.Sprintf("app=%s,component=admin", app),
+	})
+	for _, pod := range pods {
+		// Verfy that logfile nuocollector input plugin is collecting and outputing data to stdout
```

Comment:
typo in `verfy` and `outputing` here and beow.

Reply:
Thanks